riscv-regs/src/csrs/mod.rs

```diff
@@ -36,6 +36,8 @@ type VsiselectReg<R, const V: u64> = IndirectReadWriteRiscvCsr<
 >;
 
 pub struct CSR {
+    pub time: ReadWriteRiscvCsr<time::Register, CSR_TIME>,
+
```

Comment:
this is already available as `CSR.hpmcounter[1]`, no?

Reply:
Didn't notice that. Thanks :+1: 
I personally find it less confusing to have `time`, `cycle` and `insret` to be separate from `hpmcounter` csrs. I am leaving this as it is for now and revert to using `hpmcounter[1]`.